src/util/mod.rs

```diff
@@ -26,6 +26,7 @@ pub mod hash;
 pub mod iter;
 pub mod misc;
 pub mod uint;
+pub(crate) mod sha2;
```

Comment:
What does `pub(crate)` mean? Will it increase our minimum compiler version?

Reply:
Ah, I suppose it exposes this only to the entire crate rather than to downstream users?

Reply:
Yea, its pub to the crate, the min version for it appears to be 1.18, whats the min version on rust-bitcoin?

Reply:
After discussion on IRC we landed on 1.14 (currently shipping Debian version), though right now we actually don't compile on 1.14.

However with https://github.com/rust-bitcoin/rust-bech32/pull/14 and a couple minor changes which I will PR presently, we will compile on 1.14.

Reply:
I'll just make this pub, then. WIthout fuzztarget docs will just say "this reexports crypto::Sha256/Sha512", so thats fine.